docs/pages/kresponsivewindow.vue

```diff
@@ -106,10 +187,24 @@
 
 <script>
 
+  import { computed } from '@vue/composition-api';
   import responsiveWindowMixin from '~~/lib/KResponsiveWindowMixin.js';
+  import useKResponsiveWindow from '~~/lib/useKResponsiveWindow.js';
 
   export default {
     mixins: [responsiveWindowMixin],
+    setup() {
+      const {
+        windowBreakpoint: _windowBreakpoint,
+        windowIsLarge: _windowIsLarge,
+      } = useKResponsiveWindow();
+
+      const _boxStyle = computed(() => {
+        return { display: _windowIsLarge.value ? 'inline-block' : 'block' };
+      });
+
+      return { _windowBreakpoint, _windowIsLarge, _boxStyle };
+    },
     computed: {
       boxStyle() {
```

Comment:
Should you remove this computed value?

Reply:
This is used for the mixin. I will rename this so its more clear